modules/Microsoft.DataProtection/backupVaults/readme.md

```diff
@@ -14,10 +14,10 @@ This module deploys DataProtection BackupVaults.
 
 | Resource Type | API Version |
 | :-- | :-- |
-| `Microsoft.Authorization/locks` | [2017-04-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2017-04-01/locks) |
+| `Microsoft.Authorization/locks` | [2017-04-01](https://docs.microsoft.com/en-us/azure/templates) |
```

Comment:
@AlexanderSehr i think this URL is impacted by the recent doco error that was raised internally. Should we wait for that to be fixed before merging? As we might have to regenerate docs later

Reply:
We should and it is. @JPEasier please re-run the generation. The link should be back now :)

Reply:
@ahmadabdalla, @AlexanderSehr it's updated now ;)